src/shared/sources/pl/index.js

```diff
@@ -0,0 +1,84 @@
+const parse = require('../_lib/parse.js')
+const maintainers = require('../_lib/maintainers.js')
+
+const isoMap = require('./mapping.json')
+
+const country = "iso1:PL"
+
+module.exports = {
+    country,
+    maintainers: [ maintainers.qgolsteyn, maintainers.ciscorucinski ],
+    friendly: {
+        name: 'Ministry of Health of the Republic of Poland',
+        url: 'http://www.mz.gov.pl/'
+    },
+    scrapers: [
+        // TODO (scraper) PL: 2020-03-18 scraper
```

Comment:
This apparently cannot be removed since there is a historic scraper test that should run through this older scraper. However, it has 17 results (16 "states" + 1 country). However, isoMap doesn't exist for the country level

Reply:
Can omit the totals item before doing the loop through of the states.

If you like you can also use `assert-totals-are-reasonable` util to check that the total vs the summed states total are in the ballpark of eachother.

Reply:
This wasn't my original code, so I haven't really changed its structure. I can look at combining the loops. But what lines are you talking about with the totals/country item?